tests/interfaces/default_qubit_2_integration/test_jax_jit_qnode_default_qubit_2.py

```diff
@@ -46,19 +51,28 @@
 
 
 @pytest.mark.parametrize(
-    "interface,dev,diff_method,grad_on_execution", interface_and_qubit_device_and_diff_method
+    "interface,dev,diff_method,grad_on_execution, device_vjp",
+    interface_and_qubit_device_and_diff_method,
 )
 class TestQNode:
     """Test that using the QNode with JAX integrates with the PennyLane
     stack"""
 
-    def test_execution_with_interface(self, dev, diff_method, grad_on_execution, interface):
+    def test_execution_with_interface(
+        self, dev, diff_method, grad_on_execution, interface, device_vjp
+    ):
         """Test execution works with the interface"""
         if diff_method == "backprop":
             pytest.skip("Test does not support backprop")
```

Comment:
I know this was here before, but why does this test not support backprop? :thinking:

Reply:
Just ran the test, and it passes :upside_down_face: (same for non-JIT)

Reply:
I have no idea?